Wikipedia/Code/ArticleViewController+References.swift

```diff
@@ -197,3 +203,9 @@ extension ArticleViewController: ReferenceBackLinksViewControllerDelegate {
         }
     }
 }
+
+extension ArticleViewController: UIGestureRecognizerDelegate {
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        return true
+    }
+}
```

Comment:
```suggestion
```

Same note as `referenceWebViewBackgroundTapGestureRecognizer` – removing the unneeded delegate assignment removes the need for this conformance.